issues.md

```diff
@@ -1,156 +1,150 @@
+## Guidelines for triaging issues
 
-## Overall triage process
+If you are a maintainer or an owner in the MDN GitHub organization, you are responsible for triaging issues in one or more MDN repository.
 
-At a high level, the process for triage looks like so:
+The overall process for triaging includes some [general](#general_triaging_tasks) and some [issue-specific tasks](#issue_specific_triaging_tasks).
 
-Triage preparation:
+### General triaging tasks
 
-- Decide on triagers — Who will do the regular triage?
-- Set initial labels — As soon as a new bug comes in, give it the "needs-triage" label, to signify that it needs to be triaged (this should happen automatically), plus a "Content:" label to signify what topic area it is in, e.g. "Content:HTML". Anyone can do this as they spot bugs coming in, but the MDN core team will keep an active eye on this.
-- Set aside triage time — set out a regular 30-minute slot in which to do the triage, each week.
+- When an issue is opened, the `needs triage` label is set on the issue automatically. You can search for this label to look for issues that [need to be triaged](#issue_specific_triaging_tasks). Contributors or anybody else should not work on the issue until the issue has been triaged. (Triagers should remember to remove the `needs triage` label after triaging the issue.)
 
-Triage for each issue:
+- In the [mdn/content repository](https://github.com/mdn/content/issues), an additional `Content:` label, such as `Content:CSS` or `Content:WebAPI`, is set on the issue automatically. This gets set based on the MDN URL mentioned in the issue. You can use the content-specific label to look for issues to be triaged in your specific topic area.
 
-- Checklist — run through checklist to see if it is ready to triage.
-- Set priority measure — according to priority rules.
-- Provide further information to help other contributors start working on bugs more easily.
-- Set other labels — there are other labels to set, to help people to choose issues to work on.
+- If an issue concerns an active, non-en-US locale, set the appropriate label, such as `l10n-fr`, `l10n-zh`, or `l10n-ja`. The teams for those locales will pick these issues up and triage them.
 
-Check through old bugs — look at existing bugs, and see if there are any that are stalled, or need closing, etc.
+- You don't need to actively triage issues all the time. Set aside time, say 30 minutes every week, to triage issues on a regular basis in your area of responsibility. Triaging doesn't have to be done as part of a synchronous meeting or even at the same time as everyone else, but it should be done regularly to make sure that the backlog of untriaged bugs doesn't get too high.
 
-## Triage preparation
+- Apart from triaging incoming issues every week, review the list of old bugs to see if there are any that are stalled, need closing, or are no longer relevant.
```

Comment:
We can mention that these will be auto-labelled as stale after 30 days.

Reply:
Done.

The [idle workflow](https://github.com/mdn/workflows/blob/main/.github/workflows/idle.yml) does apply the `idle` label after 30 days.

Does that mean we need to update the `idle` label description? Currently it is:
`Issues and pull requests with no activity for three months. `